src/OrchardCore.Modules/OrchardCore.Media/Controllers/AdminController.cs

```diff
@@ -59,7 +59,7 @@ public async Task<IActionResult> Index()
             return View();
         }
 
-        public async Task<ActionResult<IAsyncEnumerable<IFileStoreEntry>>> GetFolders(string path)
+        public async Task<ActionResult> GetFolders(string path)
```

Comment:
Why removing the return types? wouldn't it be `Task<ActionResult<IEnumerable<IFileStoreEntry>>`

Reply:
I had simply copied the signature from the work around sample really quickly.  You're right, it's probably better to keep the return generic for things like swagger tools to use.  I'll update it.

Reply:
Updated.